File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    let answer = true;
    
    const stringArr = s.split('');
    
    //p, y 의 개수 체크
    const countP = stringArr.filter(ele => ele === 'p' || ele === 'P').length;
    const countY = stringArr.filter(ele => ele === 'y' || ele === 'Y').length;
    
    //p, y 개수 비교 -> 같으면 T, 다르면 F, 모두 0 -> T
    if(countP !== countY){
        answer = false;
    }

    return answer;
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.cpp
#include <vector>
#include <algorithm>
#include <map>
#include <queue>
#include <iostream>
#include <string>
using namespace std;

bool solution(string s)
{
    bool answer = true;

    // [실행] 버튼을 누르면 출력 값을 볼 수 있습니다.
    // cout << "Hello Cpp" << endl;
    
    int countP = count(s.begin(), s.end(), 'p') + count(s.begin(), s.end(), 'P');
    
    int countY = count(s.begin(), s.end(), 'y') + count(s.begin(), s.end(), 'Y');
    
    

    return countP == countY;
}